remote/src/main/java/org/buffer/android/boilerplate/remote/ArticleRemoteImpl.kt

```diff
@@ -15,14 +15,16 @@ import javax.inject.Inject
 class ArticleRemoteImpl @Inject constructor(private val articleService: ArticleService,
                                             private val entityMapper: ArticleEntityMapper) :
         ArticleRemote {
+    val apikey: String
 
-    override fun getArticles(): Flowable<List<ArticleEntity>> {
+    init {
         val klass = Class.forName("org.buffer.android.boilerplate.ui.BuildConfig")
```

Comment:
I see, awesome implementation, using reflection to detect the runtime package inside `mobile-ui`. But, I'll try to find an alternative, this is tends to be painful if there's a package name changed

Reply:
except this is used for `andorid dynamice feature`

Reply:
or using NDK instead to call native c++ method in java/kotlin, see here #6